director.js

```diff
@@ -30,12 +33,32 @@ function adjustVaue(amount) {
 }
 
 $(function () {
+
+    $("#player1_featuredcard, #player2_featuredcard").autocomplete({
+    minLength: 3,
+    source: function(request, response) {
+        $.get("https://api.magicthegathering.io/v1/cards?name=" + request.term,
+            function(data) {
+                response(_.uniq(_.map(data.cards, function(x) { return x.name })))
+            })
+    },
+    select: function(event, ui) {
+        firebase.database().ref($(this).attr("id").replace("_", "/")).set(ui.item.value);
+    },
+});
+
     $("#mode").change(function () {
         firebase.database().ref('mode').set($(this).val());
     });
-    $("input, textarea").change(function () {
+    // don't stomp on the autocomplete change eventhandler
+    $("input[id!='player1_featuredcard'][id!='player2_featuredcard'], textarea").change(function () {
         firebase.database().ref($(this).attr("id").replace("_", "/")).set($(this).val());
     });
+    $(".clear").click(function () {
+        var input = $(this).siblings("input");
+        input.val("");
+        firebase.database().ref(input.attr("id").replace("_", "/")).set(input.val());
```

Comment:
wait, really? I explicitly don't set the change handler for these two text boxes because that seems to stomp on autocomplete working at all (it needs its own change callback)